src/components/payment/PaymentSection.js

```diff
@@ -134,10 +136,15 @@ class PaymentSection extends React.Component {
       title = 'There is no training scheduled'
     } else {
       title = 'Standard priced ticket'
-      trainingInstanceId = training.id
+      trainingType = training.type
+
+      if (trainingType === MEETUP) {
+        eventId = training.id
+      } else {
+        trainingInstanceId = training.id
+      }
```

Comment:
Ternary rather than nested of else?

Reply:
@ric9176 we are assigning two different variables eventId and trainingInstanceId, so I think we can't use the ternary